src/utils/wslUtils.ts

```diff
@@ -0,0 +1,18 @@
+"use strict";
+
+import * as cp from "child_process";
+import * as vscode from "vscode";
+
+export function useWsl(): boolean {
+    const leetCodeConfig: vscode.WorkspaceConfiguration = vscode.workspace.getConfiguration("leetcode");
+
+    return process.platform === "win32" && leetCodeConfig.get<boolean>("useWsl") === true;
+}
+
+export function toWslPath(path: string): string {
+    return cp.execFileSync("wsl", ["wslpath", "-u", `${path.replace(/\\/g, "/")}`]).toString().trim();
```

Comment:
If we use `"` to wrap the path instead of replace `\`, will it be workable?

Reply:
Not Work :(

```js
function toWslPath(path) {
    return cp.execFileSync("wsl", ["wslpath", "-u", `"${path}"`]).toString().trim();
}
```

```
module.js:549
    throw err;
    ^

Error: Cannot find module '/mnt/y/env/VSCode-win32-x64/Y:/env/VSCode-win32-x64/.vscode/extensions/shengchen.vscode-leetcode-0.7.0/node_modules/leetcode-cli/bin/leetcode'
    at Function.Module._resolveFilename (module.js:547:15)
    at Function.Module._load (module.js:474:25)
    at Function.Module.runMain (module.js:693:10)
    at startup (bootstrap_node.js:191:16)
    at bootstrap_node.js:612:3
```

Reply:
We could focus on `cpUtils` and `workspaceUtils` provide service for others.

Reply:
@purocean That is because by default the option `shell` of `execFileSync` is false. If add `{shell: true}` to the argument list, then I think it should work.

Another thing I'm concerning is that here we use `execFileSync()`. Personally, I prefer use `async/await` wherever possible. Take a look at this: https://nodejs.org/en/docs/guides/blocking-vs-non-blocking/
So can we use executeCommand in the cpUtils?

Reply:
`{shell: true}` work perfectly.

https://github.com/purocean/vscode-leetcode/blob/d330913ad843179e12d0248a155f8a1f570e9f6f/src/shared.ts#L7

We need a lot of works if we use async/await. Abstract `leetcode` command is a better option.

But we could improve step by step. Half a loaf is better than no bread.

Reply:
OK, make sense. Tracking issue: #48